buildscripts/pages/gallery.html

```diff
@@ -75,7 +89,8 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
 				name: "Japan",
 				image: "https://models.euclideon.com/demos/thumbs/japan.png",
 				project: "udStream.html?f=euclideon:project/f4cddc9c-6922-4762-8682-94a536dac451",
-				description: "This has some high quality models from Japan."
+				description: "This has some high quality models from Japan.",
+				link: "https://www.euclideon.com/mount-fuji/"
```

Comment:
`link` isn't used anywhere in the system

Reply:
Removed link.